index.html

```diff
@@ -8,26 +8,38 @@
     <title>WiFi4Kids</title>
   </head>
   <body>
+    <div
+      id="loadOverlay"
+      style="
+        background-color: #333;
+        position: absolute;
+        top: 0px;
+        left: 0px;
+        width: 100%;
+        height: 100%;
+        z-index: 2000;
```

Comment:
To make sure it's on top 😂

Reply:
i googled for this, html was loading before css